src/main/webapp/resources/js/components/AnalysesTable/AnalysisDuration.jsx

```diff
@@ -19,15 +19,17 @@ export function AnalysisDuration({ duration, analysisId, updateDelay, state }) {
 
   // Update the analysis duration using polling
   const intervalId = useInterval(() => {
-    if(state.value !== "COMPLETED" && state.value !== "ERROR") {
-      let rowData = analysesTableContext.rows.filter(row => row.identifier === analysisId);
+    if (state.value !== "COMPLETED" && state.value !== "ERROR") {
```

Comment:
Add type definitions to the props and then clean up the JSDoc

Reply:
This file has nothing to do with my refactor. I had to update the date-utilities import and it ended up formatting the whole file